docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -176,33 +111,160 @@ This example configuration file accepts input from an OpenTelemetry Agent, proce
+
+| `secret_token` | Credential for {apm-server-ref-v}/secret-token.html[secret token authorization]. Must also be enabled in Elastic APM Server. (optional)
+
+| `ca_file` | Root Certificate Authority (CA) certificate for verifying the server's identity if TLS is enabled. (optional)
+
+| `cert_file` | Client TLS certificate. (optional)
+
+| `key_file` | Client TLS key. (optional)
+
+| `insecure` | Disable verification of the server's identity if TLS is enabled. (optional)
+
+|===
+
+[float]
+[[instrument-apps-collector]]
+===== Instrument applications
+
+To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications
+with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to execute your Java app using the
+https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry agent for Java]. 
+
+By defining the following environment variables, you can customize the OTLP endpoint the agent will use to communicate with
+APM Server.
+
+[source,bash]
+----
+export OTEL_RESOURCE_ATTRIBUTES=service.name=frontend,service.version=1.0-SNAPSHOT,deployment.environment=staging
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://apm_server_url:8200
+java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
+     -jar target/frontend-1.1.jar
+----
+
+|=== 
+
+| `OTEL_RESOURCE_ATTRIBUTES` | The service name to identify your application.
+
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | APM Server URL. The host and port that APM Server listens for events on.
+
+|===
+
+You are now ready to collect <<open-telemetry-elastic-traces-metrics,traces and metrics>>, <<open-telemetry-elastic-verify,verifying metrics>>,
+and <<open-telemetry-elastic-kibana,visualizing metrics>> in {kib}.
 
-NOTE: For more information about getting started with an OpenTelemetry Collector,
-see the {ot-collector}[OpenTelemetry collector] docs.
+[[open-telemetry-elastic-protocol]]
+==== APM Server native support of OpenTelemetry protocol
+
+experimental::[This feature is experimental and may be changed in a future release. It is only available in a self-managed environment.]
+
+The APM Server native support of OpenTelemetry protocol integration allows you to send collected telemetry data directly from your applications to APM Server.
+Trace data collected from your services and the metrics data collected from your applications and infrastructure are sent using the
+OpenTelemetry protocol.
+
+image::images/open-telemetry-protocol-arch.png[OpenTelemetry Elastic protocol architecture diagram]
 
 [float]
-[[open-telemetry-elastic-caveats]]
-==== Caveats
+[[instrument-apps-apm-server]]
+===== Instrument applications
+
+To export traces and metrics to APM Server, ensure that you have instrumented your services and applications
+with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to execute your Java app using the
```

Comment:
`execute` again. It's possible this word is correct and I just haven't heard it yet.

Reply:
`Instrument` is a much better word here.